rskj-core/src/main/java/co/rsk/peg/BridgeUtils.java

```diff
@@ -114,22 +112,28 @@ public static boolean isValidLockTx(BtcTransaction tx) {
      * It checks if the tx doesn't spend any of the federations' funds and if it sends more than
      * the minimum ({@see BridgeConstants::getMinimumLockTxValue}) to any of the federations
      * @param tx the BTC transaction to check
-     * @param federations the active federations
+     * @param activeFederations the active federations
+     * @param retiredFederationP2SHScript the retired federation P2SHScript. Could be {@code null}.
      * @param btcContext the BTC Context
      * @param bridgeConstants the Bridge constants
      * @return true if this is a valid lock transaction
      */
-    public static boolean isLockTx(BtcTransaction tx, List<Federation> federations, Context btcContext, BridgeConstants bridgeConstants) {
+    public static boolean isLockTx(BtcTransaction tx, List<Federation> activeFederations, Script retiredFederationP2SHScript,
```

Comment:
It could be useful to rename this method to `isPegInTx` so that expression `lock` is no longer used